lib/ash/resource/change/builtins.ex

```diff
@@ -286,6 +286,27 @@ defmodule Ash.Resource.Change.Builtins do
     {Ash.Resource.Change.Select, target: value, ensure?: true}
   end
 
+  @doc """
+  Cascade this resource's destroy action to the related resource's destroy action.
+
+  ## Example
+
```

Comment:
Is it worth pointing out somewhere that this might not work with foreign key constraints in ash_postgres? You might need to defer constraints to use this strategy. And also if you're using ash_postgres you'd probably prefer the automatic cascade using PG?